docs/conf/portland/2022/news/announcing-schedule.rst

```diff
@@ -0,0 +1,77 @@
+:template: {{year}}/generic.html
+
+.. post:: Apr 11, 2022
+   :tags: {{year}}, portland-{{year}}, schedule
+
+Announcing Full Schedule
+========================
+
+Hey everyone!
+
+We're just over a month away from the online Portland conference, and we're working hard to make sure it'll be as welcoming and as fun as the in-person event you're probably familiar with.
+
+Today we're announcing the full presentation schedule and the platform we'll be hosting the conference on.
+We are working on last-minute details for a few more activities during the conference, so look out for more information in our next mailing.
+
+Conference platform
+--------------------
+
+Following the success of last year's virtual format, the conference will again be held on https://hopin.to/. Hopin not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and expo booths. We're updating our Welcome Wagon guide, to be released soon, so it'll all feel familiar before you've even arrived.
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the :doc:`/conf/portland/{{year}}/schedule/` for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. We're excited to continue the new tradition of live Q&A after most talks, this year two speakers will be chatting with the MC after every couple of talks, a little like a mini panel, which will come from your questions but be moderated by our MC.
```

Comment:
@ericholscher will this convert ok in the mail out? `:doc:`/conf/portland/{{year}}`

Reply:
Yea, we make all links fully qualified (99% sure :)